eng/pipelines/test-integration-helix.yml

```diff
@@ -0,0 +1,126 @@
+parameters:
+  - name: poolName
+    type: string
+  - name: queueName
+    type: string
+  - name: configuration
+    type: string
+    default: 'Debug'
+  - name: timeout
+    type: number
+  - name: testRuns
+    type: object
+    default:
+      - oop64bit: true
```

Comment:
the template accepts a list of input parameters so that we can re-use the build assets for multiple different configurations of test run.